prov/sm2/src/sm2_ep.c

```diff
@@ -228,6 +228,7 @@ static void cleanup_shm_resources(struct sm2_ep *ep)
 		} else {
 			/* TODO Tell other side that we haven't processed their
 			 * message, just returned xfer_entry */
+			xfer_entry->hdr.proto_flags &= ~SM2_GENERATE_COMPLETION;
```

Comment:
Maybe these patches are out of order and this will become clear in the next commits but in the previous commit you added the flag but never use it and in this one you just unset it?

Reply:
Squashed this commit into the previous one